lib/streamlit/__init__.py

```diff
@@ -186,6 +186,7 @@ def _update_logger() -> None:
 write_stream = _main.write_stream
 color_picker = _main.color_picker
 status = _main.status
+popover = _main.popover
```

Comment:
nit: It looks like the list above is kind of sorted alphabetically, so it might make sense to sort it into

Reply:
Moved it up, but I think this sorting is more of an artifact of the past. There isn't really any reasonable argument why it needs to be sorted.

Reply:
> There isn't really any reasonable argument why it needs to be sorted.

I feel having a sorted list helps to find things or to prevent having duplicate assignments or similar scenarios.